pipeline/terraform0.12/stack/security_groups.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_security_group" "service_egress" {
+  name        = "${local.namespace_hyphen}_service_egress"
+  description = "Allow traffic between services"
+  vpc_id      = var.vpc_id
+
+  egress {
+    from_port = 0
+    to_port   = 0
+    protocol  = "-1"
+
+    cidr_blocks = [
+      "0.0.0.0/0",
+    ]
+  }
+}
+
+resource "aws_security_group" "interservice" {
+  name        = "${local.namespace_hyphen}_interservice"
+  description = "Allow traffic between services"
```

Comment:
The egress and interservice security groups have the same description. Is this a copy/paste error?

Reply:
Twas indeed, not sure where from

Reply:
Changed - thanks!